### ISSUE TITLE
problems resolving testng with Kobalt .919

### ISSUE THREAD

**Issue by pabl0rg** [cbeust/kobalt#321]
I initialized a new project with the kotlin template and then tried to assemble it.  Kobalt fails to resolve testng.

I also had to do dos2unix to get the line-endings right.

```
Juans-MacBook-Pro lt-reports-ktor> chmod +x ~/.kobalt/wrapper/dist/kobalt-0.919/bin/kobaltw
Juans-MacBook-Pro lt-reports-ktor> dos2unix ~/.kobalt/wrapper/dist/kobalt-0.919/bin/kobaltw
dos2unix: converting file /Users/juanliska/.kobalt/wrapper/dist/kobalt-0.919/bin/kobaltw to Unix format...
Juans-MacBook-Pro lt-reports-ktor> ~/.kobalt/wrapper/dist/kobalt-0.919/bin/kobaltw --init kotlin
            _  __          _               _   _
           | |/ /   ___   | |__     __ _  | | | |_
           | ' /   / _ \  | '_ \   / _` | | | | __|
           | . \  | (_) | | |_) | | (_| | | | | |_
           |_|\_\  \___/  |_.__/   \__,_| |_|  \__|  0.919



Template "kotlin" installed
Now you can run either `./kobaltw test` or `./kobaltw run`
Juans-MacBook-Pro lt-reports-ktor> ./kobaltw assemble
              __ __           __              __   __
             / //_/  ____    / /_   ____ _   / /  / /_
            / ,<    / __ \  / __ \ / __ `/  / /  / __/
           / /| |  / /_/ / / /_/ // /_/ /  / /  / /_
          /_/ |_|  \____/ /_.___/ \__,_/  /_/   \__/  0.919

***** WARNING Invoking repos() directly is deprecated, use the buildScript{} directive
***** WARNING Invoking repos() directly is deprecated, use the buildScript{} directive
Parallel build starting
     ╔══════════════════════════╗
     ║ Building lt-reports-ktor ║
     ╚══════════════════════════╝
───── lt-reports-ktor:compile
*****
***** ERROR Error: com.beust.kobalt.KobaltException: Couldn't resolve org.testng:testng:jar:6.10.1-SNAPSHOT
*****

Thread report
╔════════════════════════════════════════╗
║  Time (sec) ║ Thread 13                ║
╠════════════════════════════════════════╣
║  0          ║ lt-reports-ktor          ║
╚════════════════════════════════════════╝
PARALLEL BUILD SUCCESSFUL (0 SECONDS), sequential build would have taken 0 seconds
```

**Comment by cbeust** [comment]
Can you `rm -rf ~/.kobalt/cache/org/testng` and try again?

**Comment by pabl0rg** [comment]
```
Juans-MacBook-Pro lt-reports-ktor> rm -rf ~/.kobalt/cache/org/testng
Juans-MacBook-Pro lt-reports-ktor> kbw assemble
            _  __          _               _   _
           | |/ /   ___   | |__     __ _  | | | |_
           | ' /   / _ \  | '_ \   / _` | | | | __|
           | . \  | (_) | | |_) | | (_| | | | | |_
           |_|\_\  \___/  |_.__/   \__,_| |_|  \__|  0.919

***** WARNING Invoking repos() directly is deprecated, use the buildScript{} directive
***** WARNING Invoking repos() directly is deprecated, use the buildScript{} directive
Parallel build starting
     ╔══════════════════════════╗
     ║ Building lt-reports-ktor ║
     ╚══════════════════════════╝
───── lt-reports-ktor:compile
Downloaded artifact org.testng:testng:pom:6.10.1-20170205.203018-17 from  (http://repository.jetbrains.com/all/, default, releases+snapshots)
*****
***** ERROR Error: com.beust.kobalt.KobaltException: Couldn't resolve org.testng:testng:jar:6.10.1-SNAPSHOT
*****
```

**Comment by cbeust** [comment]
Ok I found out that maven-aether downloads snapshots even from repositories I configured as "release" only, so I must be misunderstanding how this works.

It might actually be correct: http://stackoverflow.com/questions/4407055/how-do-i-stop-maven-version-ranges-from-using-snapshots

**Comment by pabl0rg** [comment]
no wonder maven intimidates a lot of folks

**Comment by cbeust** [comment]
This should be fixed in 0.921, can you try again?
